src/core/server/index.ts

```diff
@@ -291,6 +298,7 @@ export interface RequestHandlerContext {
     rendering: IScopedRenderingClient;
     savedObjects: {
       client: SavedObjectsClientContract;
+      typeRegistry: ISavedObjectTypeRegistry;
     };
```

Comment:
This is going to be used for the next step of the migration (migrate the server routes in new plugin). I could retrieve the registry using `getStartServices`, but I thought it made sense to expose it there. Tell me if I should remove it.

Reply:
Makes sense to me to have it here 👍